dxr/plugins/rust/tests/__init__.py

```diff
@@ -8,8 +15,10 @@ class RustSingleFileTestCase(SingleFileTestCase):
     @classmethod
     def config_input(cls, config_dir_path):
         config = super(RustSingleFileTestCase, cls).config_input(config_dir_path)
-
+        
         config['DXR']['enabled_plugins'] = 'pygmentize rust'
         config['code']['build_command'] = '$RUSTC mod.rs'
+        
+        os.environ["RUSTC"] = "rustc -Zsave-analysis"
```

Comment:
It's weird that we do this in config_input() here and in setup_class() everywhere else. Also, can we factor up at least the string "rustc -Zsave-analysis" or perhaps the whole operation?